Analysis/include/SummaryTuple.h

```diff
@@ -23,16 +23,21 @@ This file is part of https://github.com/hh-italian-group/h-tautau. */
     VAR(std::vector<UInt_t>, triggerFilters_triggerIndex) \
     VAR(std::vector<UInt_t>, triggerFilters_LegId) \
     VAR(std::vector<std::string>, triggerFilters_name) \
-    /* MC truth event splitting */ \
-    VAR(std::vector<UInt_t>, lhe_n_partons) \
-    VAR(std::vector<UInt_t>, lhe_n_b_partons) \
-    VAR(std::vector<UInt_t>, lhe_ht10_bin) \
-    VAR(std::vector<ULong64_t>, lhe_n_events) \
     /* Skimmer Variables */\
     VAR(std::vector<UInt_t>, file_desc_id) /* vector of File id in TupleSkimmer. */ \
     VAR(std::vector<std::string>, file_desc_name) /* vector of File name in TupleSkimmer. */ \
     VAR(UInt_t, n_splits) /* Number of splits for a file in TupleSkimmer. */ \
     VAR(UInt_t, split_seed) /* Seed for splitting in TupleSkimmer. */ \
+    /* MC truth event splitting */ \
+    VAR(std::vector<UInt_t>, lhe_n_partons) \
+    VAR(std::vector<UInt_t>, lhe_n_b_partons) \
+    VAR(std::vector<UInt_t>, lhe_ht10_bin) \
+    VAR(std::vector<ULong64_t>, lhe_n_events) \
+    /* Top reweighting */ \
+    VAR(Float_t, genEventWeight) /* gen event weight */ \
+    VAR(Float_t, gen_top_pt) /* pt of gen ME top */ \
+    VAR(Float_t, gen_topBar_pt) /* pt of gen ME anti-top */ \
+    VAR(Int_t,   genEventType) /* top gen event type */ \
```

Comment:
Branches genEventWeight, gen_top_pt and gen_topBar_pt are not needed.
Should be 2 branches:
```c++
VAR(std::vector<Int_t>, genEventType)
VAR(std::vector<ULong64_t>, genEventType_n_events)
```
All methods to merge check SummaryTuple should be adjusted accordingly.